database.go

```diff
@@ -1139,9 +1137,9 @@ func (db *database) dropMeasurement(name string) error {
 }
 
 // dropSeries will delete all data with the seriesID
-func (rp *RetentionPolicy) dropSeries(seriesID uint32) error {
+func (rp *RetentionPolicy) dropSeries(seriesIDs ...uint32) error {
```

Comment:
I'm curious about this style. More preferred/idiomatic than doing `[]uint32`? Or is it because you want to be able to support just passing in one ID?